### ISSUE TITLE
`prototype` does not recognize a branch

### ISSUE THREAD

**Issue by sergej-koscejev** [git-town/git-town#3865]
### Steps to reproduce

I'm not sure how I got here.

### Command output

```shell
(migrate-task-20223) $ git config git-town.prototype-branches
... migrate-task-20223 ... migrate-task-20223 migrate-task-20223 migrate-task-20223 migrate-task-20223

(migrate-task-20223) $ git-town prototype migrate-task-20223

Error: there is no branch "migrate-task-20223"

(migrate-task-20223) $ git-town prototype
branch "migrate-task-20223" is now a prototype branch

(migrate-task-20223) $ git config git-town.prototype-branches
... migrate-task-20223 ... migrate-task-20223 migrate-task-20223 migrate-task-20223 migrate-task-20223 migrate-task-20223
```

### Additional information

Basically, `git-town prototype` doesn't see the current branch if I refer to it by name, but is happy to add it multiple times to the list if it's the current branch.

**Comment by kevgo** [comment]
Thanks for reporting this! I cannot reproduce the problem using the steps you provided, i.e. the command works for me. Can you please post the output of `git-town prototype migrate-task-20223 --verbose` (minus confidential data)?

**Comment by kevgo** [comment]
I have re-created the scenario you outlined in https://github.com/git-town/git-town/blob/main/features/prototype/give_no_branch/on_prototype_branch.feature. It works now. Closing this ticket. If you still run into this problem after the next release, please comment here.

**Comment by kevgo** [comment]
I have released [v15.2](https://github.com/git-town/git-town/releases/tag/v15.2.0) with some bugfixes in the affected area. Hopefully it fixes the issue you encountered here. Please take it for a spin!
